examples/projects/NUCLEO-F401RE/led/lib/Led/led.c

```diff
@@ -54,4 +58,8 @@ static void Led_MsgHandler(service_t *service, msg_t *msg)
             HAL_GPIO_WritePin(LED_GPIO_Port, LED_Pin, Led_last_state);
         }
     }
+    if (msg->header.cmd == WRITE_ALIAS)
```

Comment:
I don't understand how you manage that? How the command WRITE_ALIAS is treat by luos and by the service?

Reply:
@nicolas-rabault